### ISSUE TITLE
Solution for not overlapping content

### ISSUE THREAD

**Issue by gjm** [wire-elements/wire-spy#20]
Thanks for reporting! This is now fixed https://github.com/wire-elements/wire-spy/commit/f7b2b843b611e8f7dd757c2dadec9becf5bb383b

_Originally posted by @PhiloNL in https://github.com/wire-elements/wire-spy/issues/14#issuecomment-2410519586_
           
This issue is still present for me...

**Comment by PhiloNL** [comment]
Can you take a look with your devtools and see if the padding is added to your body? It should offset the height of the toolbar:

![image](https://github.com/user-attachments/assets/1f2db637-6ee8-411e-be8c-8c65acc3685d)

**Comment by gjm** [comment]
Yes it is added. I have inner div's with `height: 100% !important`. Maybe those are breaking wire-spy's layout... :disappointed:

**Comment by PhiloNL** [comment]
Hmm not sure how to solve this at the moment. Open to PR's or suggestions.
